crypto/rsa/rsa_lib.c

```diff
@@ -774,6 +774,12 @@ int rsa_get0_all_params(RSA *r, STACK_OF(BIGNUM_const) *primes,
     if (r == NULL)
         return 0;
 
+    pnum = RSA_get_multi_prime_extra_count(r);
```

Comment:
looks like this line needs to be removed..

Reply:
Fixed